<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="che" failures="1" tests="4" name="Test in edge" time="8.184" errors="0" timestamp="2025-01-17T22:47:15 IST">
  <testcase classname="testCases.TC_01_add" name="AddDecimalTwoPointTwoMinusOne" time="0.839"/>
  <testcase classname="testCases.TC_01_add" name="AddZeroAndOne" time="0.576"/>
  <testcase classname="testCases.TC_01_add" name="verifyAddition" time="0.516">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at testCases.TC_01_add.verifyAddition(TC_01_add.java:35)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 19 stack frames]]>
    </failure>
  </testcase> <!-- verifyAddition -->
  <testcase classname="testCases.TC_01_add" name="verifySubtraction" time="0.562"/>
</testsuite> <!-- Test in edge -->
